doc/verilog_systemverilog.txt

```diff
@@ -448,6 +448,19 @@ Example:
     Note: The commands |:VerilogFoldingAdd| and |:VerilogFoldingRemove| are
     provided to allow an easier management of this variable.
 
+                                                      *g:verilog_quick_syntax*
+g:verilog_quick_syntax~
+Default: undefined
+
+    Whena enabled, syntax regions will not be defined with Vim's 'syntax'
+    command. This can help performance if you are using an old machine or
+    veiwing a large file such as a netlist. This can be useful if you don't
+    care about automatic indentation but still wish to have syntax coloring.
```

Comment:
In addition, it 'may' be possible to improve syntax region performance by defining them more robustly and making better use of options like `contains`, `contained`, `contained`, `nextgroup` etc. At the moment every region can contain anything which may attribute to the slow performance.